packages/cubejs-testing/test/__snapshots__/smoke-questdb.test.ts.snap

```diff
@@ -3,7 +3,7 @@
 exports[`questdb query measure: query 1`] = `
 Array [
   Object {
-    "Orders.totalAmount": "114",
+    "Orders.totalAmount": "1700",
```

Comment:
Somewhat nervous, this test snapshot changed without an obvious corresponding change in the test spec. Is this intended?

Reply:
This change has to do with QuestDB version upgrade in integration tests. There was a bug in the query engine around aggregated over UNION ALL which was fixed recently. If you check `packages/cubejs-testing/birdbox-fixtures/materialize/schema/Orders.js`, you'll notice that 1700 is the correct value.